src/lib/config/BuildConfig.ts

```diff
@@ -83,6 +90,28 @@ export class BuildConfig {
         this.win.productVersion = this.win.productVersion ? this.win.productVersion : pkg.version;
         this.win.fileVersion = this.win.fileVersion ? this.win.fileVersion : this.win.productVersion;
 
+        for (let config of [this.win, this.mac]) {
+            config.signing.cliArgsInterpolated = config.signing.cliArgs.split(' ');
+            if (config.signing.cliArgsVarsFile) {
+                const {cliArgsVarsFile} = config.signing;
+                const cliArgsVarsFileAbsolutePath = resolve(process.cwd(), cliArgsVarsFile);
+                try {
+                    const actualArgsVarsFile = readFileSync(cliArgsVarsFileAbsolutePath);
+                    const envVars = dotenv.parse(actualArgsVarsFile);
+                    const interpolatedCliArgs = config.signing.cliArgsInterpolated.map(
+                        (arg) => parseTmpl(arg, envVars));
+                    debug(`using interpolatedCliArgs CLI args: `, interpolatedCliArgs);
+                    config.signing.cliArgsInterpolated = interpolatedCliArgs;
+                } catch (err) {
+                    if (err.code === 'ENOENT') {
+                        debug(`args vars file not found: ${cliArgsVarsFile}`);
+                    } else {
+                        throw err;
+                    }
+                }
+            }
+        }
+
```

Comment:
We needed to be able to pass a password stored in a `.env` file to the key. That's the point of this block of code.